### DISCUSSION TITLE
Preservation of file permissions (*nix)

### DISCUSSION THREAD

**Discussion by STaylor135** [rejetto/hfs#742] [Ideas]
Sorry, this is kind of a bug report / feature request. 

I set up a server and had a buddy send me an app, but they went to the application folder on MacOS, select the .app "file," the site correctly detected that it's actually a directory and zipped it, then sent it.  

The problem is that the *nix file permissions aren't saved so inside that directory will be a file that needs to be executed but when placed on a new system, you'll discover that the binary doesn't have the executable permission. All you get is a generic error that it can't be ran.

**Comment by rejetto** [comment]
hey, that doesn't seem to be possible.
The browser, when the user picks the file, doesn't let me know the permissions.

**Reply by STaylor135** [reply]
Oh darn.  I'll try to remember this for the future.  What about a work around, like when selecting directories, a reminder that permissions won't be kept? 

Thank you and awesome updates!
